tools/pythonpkg/tests/conftest.py

```diff
@@ -2,15 +2,56 @@
 import os
 import pytest
 import shutil
-
+from os.path import abspath
+import glob
 import duckdb
 
 @pytest.fixture(scope="function")
 def duckdb_empty_cursor(request):
     connection = duckdb.connect('')
     cursor = connection.cursor()
     return cursor
+ 
+
+@pytest.fixture(scope="function")
+def require():
+    def _require(extension_name):
```

Comment:
Really like the require fixture! There is a little bit of duplicate code now between this fixture and the `test_extensions.py` test. It would be nice to have this in a single place maybe? I think the paths we want to search for the `require` fixture and the `test_extensions.py` test are the same so it would be nice to share that